src/releasers/go-yoshi.ts

```diff
@@ -187,4 +204,26 @@ export class GoYoshi extends ReleasePR {
   protected defaultInitialVersion(): string {
     return '0.1.0';
   }
+
+  static tagSeparator(): string {
```

Comment:
This should be moved the the sub-module code I think?

Reply:
I think we want it in the top level releaser, because this is what the step that creates GitHub tags looks at.